waterfall.js.org

waterblock79 opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (changes requested):
Hi there!

The site you have linked to for your js.org domain request returns a 404: https://waterblock79.github.io/WaterfallProject
Please fix this before we can process your js.org domain request. 🛠

waterblock79 commented:
> Hi there!
> 
> The site you have linked to for your js.org domain request returns a 404: https://waterblock79.github.io/WaterfallProject
> Please fix this before we can process your js.org domain request. 🛠

Sorry I forget to change that domain after I changed my repository name, I have fixed it a few minuted ago.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```waterfall.js.org``` to make it work. (atm it says "mincraft.js.org" ? https://github.com/waterblock79/waterfall/blob/gh-pages/CNAME)
I'll merge after we have "contact".

waterblock79 commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `waterfall.js.org` to make it work. (atm it says "mincraft.js.org" ? https://github.com/waterblock79/waterfall/blob/gh-pages/CNAME)
> I'll merge after we have "contact".

fixed. Im so sorry to make that to much mistakes, im not sure create cname or not, so I reference michaljaz/web-minecraft and copyed CNAME from this repo, but forgotten changed the domain.

indus commented:
Welcome https://waterfall.js.org
